docs/design/coreclr/botr/guide-for-porting.md

```diff
@@ -417,8 +417,6 @@ Here is an annotated list of the stubs implemented for Unix on Arm64.
     locations for NullReferenceExceptions to be generated out of a SIGSEGV
     signal.
 
```

Comment:
The whole "Some helpers are written ..." paragraph should be deleted. The helpers that this talks about are gone with this PR.